src/Mooltipass/MPNodeBLE.cpp

```diff
@@ -0,0 +1,111 @@
+#include "MPNodeBLE.h"
+
+
+MPNodeBLE::MPNodeBLE(const QByteArray &d, QObject *parent, const QByteArray &nodeAddress, const quint32 virt_addr)
+    : MPNode(d, parent, nodeAddress, virt_addr)
+{
+}
+
+MPNodeBLE::MPNodeBLE(QObject *parent, const QByteArray &nodeAddress, const quint32 virt_addr)
+    : MPNode(parent, nodeAddress, virt_addr)
+{
+}
+
+bool MPNodeBLE::isDataLengthValid() const
+{
+    return isValid();
+}
+
+bool MPNodeBLE::isValid() const
+{
+    if (NodeUnknown == getType())
+    {
+        return false;
+    }
+
+    return (data.size() == PARENT_NODE_LENGTH || data.size() == CHILD_NODE_LENGTH);
```

Comment:
ahaha I see that working given how I send data... but better look at the flags to know which data type it is: 
https://github.com/mooltipass/minible/blob/master/source_code/main_mcu/src/COMMS/comms_hid_msgs.c#L443

Reply:
I have checked this from the database description: https://mooltipass.github.io/minible/database_model
According that parent nodes are 264B and Child nodes are 528B

Reply:
should I check the type here beside the size?

Reply:
you need to check the size for a given node type. if the node is a child, you need to make sure it is 528B for example

Reply:
added the check in the latest commit